docs/question.md

```diff
@@ -0,0 +1,493 @@
+        "reputation": 1,
+        "user_id": 58519,
+        "user_type": "registered",
+        "profile_image": "https://lh5.googleusercontent.com/-kVyGXuaKbiQ/AAAAAAAAAAI/AAAAAAAAAAA/ACevoQM6oMuunuZdDgfqW9iGK8svF3jnmA/mo/photo.jpg?sz=128",
+        "display_name": "Bhathiya Wijesinghe",
+        "link": "https://stackapps.com/users/58519/bhathiya-wijesinghe"
+      },
+      "is_answered": false,
+      "score": -1,
+      "last_activity_date": 1552286273,
+      "creation_date": 1550134853,
+      "last_edit_date": 1552286273,
+      "question_id": 8236,
+      "link": "https://stackapps.com/questions/8236/urlconnection-does-not-decompress-gzip",
+      "title": "URLConnection does not decompress Gzip"
+    }
+  ],
+  "has_more": false,
+  "quota_max": 10000,
+  "quota_remaining": 9963
+}
+```
+
+Following is a sample REST request that can be handled by the upvoteQuestionById operation.
+
+**Related StackExchange API documentation**
+[https://api.stackexchange.com/docs/upvote-question](https://api.stackexchange.com/docs/upvote-question)
+
+### Downvoting an question
+Downvote an existing question.
+
+**downvoteQuestionById**
+```xml
+<stackexchange.downvoteQuestionById>
+    <id>{$ctx:id}</id>
+    <preview>{$ctx:preview}</preview>
+    <filter>{$ctx:filter}</filter>
+</stackexchange.downvoteQuestionById>
+```
+
+**Properties**
+* id: Id of the question. e.g. 8940
+* filter: Filter you have chosen to filter fields in the response. e.g. default
+* preview: Preview parameter is used for development and testing purposes. When preview is set to true,
+           request will only simulate whether it is a valid request or not and will not change
+           anything on the site.
+
+**Sample request**
+```json
+{
+  "id": "8236",
+  "preview": true
+}
+```
+
+**Sample response**
+```json
+{
+  "items": [
+    {
+      "tags": [
+        "bug",
+        "library"
+      ],
+      "owner": {
+        "reputation": 1,
+        "user_id": 58519,
+        "user_type": "registered",
+        "profile_image": "https://lh5.googleusercontent.com/-kVyGXuaKbiQ/AAAAAAAAAAI/AAAAAAAAAAA/ACevoQM6oMuunuZdDgfqW9iGK8svF3jnmA/mo/photo.jpg?sz=128",
+        "display_name": "Bhathiya Wijesinghe",
+        "link": "https://stackapps.com/users/58519/bhathiya-wijesinghe"
+      },
+      "is_answered": false,
+      "score": -1,
+      "last_activity_date": 1552286273,
+      "creation_date": 1550134853,
+      "last_edit_date": 1552286273,
+      "question_id": 8236,
+      "link": "https://stackapps.com/questions/8236/urlconnection-does-not-decompress-gzip",
+      "title": "URLConnection does not decompress Gzip"
+    }
+  ],
+  "has_more": false,
+  "quota_max": 10000,
+  "quota_remaining": 9963
+}
+```
+
+Following is a sample REST request that can be handled by the downvoteQuestionById operation.
+
+**Related StackExchange API documentation**
+[https://api.stackexchange.com/docs/downvote-question](https://api.stackexchange.com/docs/downvote-question)
```

Comment:
Add Sample configuration for one operation per resource
https://github.com/wso2-extensions/esb-connector-googlespreadsheet/blob/master/docs/workWithSpreadsheet.md#sample-configuration

Reply:
Fix with dd183b6